baseplate/lib/metrics.py

```diff
@@ -62,7 +62,17 @@
 
 
 def _metric_join(*nodes: bytes) -> bytes:
-    return b".".join(node.strip(b".") for node in nodes)
+    return b".".join(node.strip(b".") for node in nodes if node)
+
+
+def _format_tags(tags: Optional[Dict]) -> Optional[bytes]:
+    if not tags:
+        return None
+
+    parts = [""]
```

Comment:
🔕  does this start with an empty string so there's a `","` at the beginning of the string?  It might be clearer just to return `b"," + ",".join(parts).encode()`.  It would be good to have a comment explaining this if this is the approach we take so it's clear what's happening.

Reply:
That is indeed why, I wasn't sure which was cleaner